doc/refman/RefMan-ssr.tex

```diff
@@ -0,0 +1,4687 @@
+
+\newcommand{\iitem}{{\it i-item}}
+\newcommand{\ditem}{{\it d-item}}
+\newcommand{\optional}[1]{{\it[}#1{\it]}}
+\newcommand{\optsep}{{\it|}}
+
+\newenvironment{new}%
+  {\begin{Sbox}\begin{minipage}{0.97\textwidth}%
+    \begin{flushright}\textcolor{red}{\fbox{Version 1.3}}%
+      \end{flushright}\noindent}%
+  {\end{minipage}\end{Sbox}\noindent\doublebox{\TheSbox}}
+\section{Introduction}\label{sec:intro}
+
+This chapter describes an extension to the \Coq{} tactic language,
+orginally designed to provide support at the tactic level for the
+so-called \emph{small scale reflection} proof methodology.
+\marginpar{Provide a citation here?} However, its tactics and tacticals
+also provide features of general interest for a wide audience of users
+like:
+\begin{itemize}
+\item better support for forward steps
+\item common support for bookkeeping in all tactics
+\item common support for subterm selection in all tactics
+\item a unified interface for rewriting, definition expansion, and
+  partial evaluation
+\item support for so-called reflection steps.
+\end{itemize}
+
+
+In fact, only the last functionality is specific to small-scale
+reflection. All the others are of general use. Moreover, these
+features are implementing by extending the functionalities of existing
+tactic, so that in the end proof scripts shall only feature
+combinations of a very small number tactics with their intro-patterns
+and switches.
+\marginpar{Say something about the possibility to use a few features only?}
+
+\subsection*{How to read this documentation}
+
+
+The syntax of the tactics is presented as follows:
+\begin{itemize}
+\item \ssrC{terminals} are in typewriter font and \ssrN{non terminals} are
+  between angle brackets.
+\item Optional parts of the grammar are surrounded by \optional{ }
+  brackets. These should not be confused with verbatim brackets
+  \ssrC{[ ]}, which are delimiters in the \ssr{} syntax.
+\item A vertical rule \optsep indicates an alternative in the syntax, and
+  should not be confused with a
+  verbatim vertical rule between verbatim brackets \ssrC{[ | ]}.
+\item A non empty list of non terminals (at least one item should be
+  present) is represented by \ssrN{non terminals}$^+$. A possibly empty
+  one is represented by \ssrN{non terminals}$^*$.
+\item In a non empty list of non terminals, items are separated by blanks.
+\end{itemize}
+\marginpar{check this makes sense wrt Coq's manual}
+
+% Hevea has no colors
+\ifhevea \else
+\noindent We follow the default color scheme of the \ssr{} mode for
+ProofGeneral provided in the distribution:
+
+\centerline{
+\textcolor{dkblue}{\texttt{tactic}} or \textcolor{dkviolet}{\tt
+  Command} or \textcolor{dkgreen}{\tt keyword} or
+\textcolor{dkpink}{\tt tactical}}
+
+\noindent Closing tactics/tacticals like \ssrC{exact} or \ssrC{by} (see section
+\ref{ssec:termin}) are in red.
+\fi
+
+\subsection*{Acknowledgments}
+The authors would like to thank Frédéric Blanqui, François Pottier
```

Comment:
According to `file -I`, the encoding of this file is `iso-8859-1` while all the other refman files are either `us-ascii` or `utf-8`.